index.bs

```diff
@@ -1408,7 +1445,54 @@ partial interface MLGraphBuilder {
       }
     }
     </pre>
-    </div>
+</div>
+
+To <dfn>check clamp options</dfn> given |options|, run the following steps:
+    1. If |options| is not an object that [=implements=] {{MLClampOptions}}, then return `false`.
+    1. If |options|.{{MLClampOptions/minValue}} and |options|.{{MLClampOptions/maxValue}} are not a [=numeric type=], then then return `false`.
+    1. If |options|.{{MLClampOptions/minValue}} is greater than |options|.{{MLClampOptions/maxValue}}, then return `false`.
+    1. Return `true`.
+
+#### The {{MLGraphBuilder/clamp(operand, options)}} method #### {#api-mlgraphbuilder-clamp-operand-options}
+<div class="note">
+    **Arguments:**
+        - *operand*: an {{MLOperand}}. The input tensor.
+        - *options*: an optional {{MLClampOptions}}. The optional parameters of the operation.
+            - *minValue*: a {{float}} scalar. Specifies the minimum value of the range. When it is not specified, the clamping is not performed on the lower limit of the range.
+            - *maxValue*: a {{float}} scalar. Specifies the maximum value of the range. When it is not specified, the clamping is not performed on the upper limit of the range.
+    **Returns:**
+        - an {{MLOperand}}. The output tensor of the same shape as *operand*.
+</div>
+<div algorithm=clamp>
+    The {{MLGraphBuilder/clamp(operand, options)}} method steps are:
```

Comment:
`operand` is commonly used, e.g. for MLOperand or platform operand etc. Should the variable name be more specific, like `input` or `x`?

Reply:
Oh, so that was the reason for using `x`. Well, `input` is also a loaded name :). 
Since it is an MLOperand, I think using `operand` is just fine.